libs/ui-lib/lib/ocm/components/clusterWizard/CredentialsDownload.tsx

```diff
@@ -49,8 +53,24 @@ const CredentialsDownload: React.FC<{ cluster: Cluster }> = ({ cluster }) => {
 
   const downloadSingleFile = async (fileName: string) => {
     try {
-      const response = await ClustersAPI.downloadClusterCredentials(cluster.id, fileName);
-      downloadFile('', response.data, fileName);
+      if (fileName === 'kubeconfig') {
+        const response = await ClustersAPI.downloadClusterCredentials(cluster.id, fileName);
+        downloadFile('', response.data, fileName);
+        return true;
+      }
+
+      await ClustersAPI.downloadClusterCredentials(cluster.id, 'kubeadmin-password');
+      const file = new Blob(
+        [
+          `password: ${credentials?.password || ''} \n`,
+          `username: ${credentials?.username || ''} \n`,
+          `console-url: ${credentials?.consoleUrl || ''}`,
+        ],
```

Comment:
Lets use `js-yaml` to convert js object to yaml.

```suggestion
      const credentials = {
        username: credentials?.password || ''
        password: credentials?.username || ''
        console-url: credentials?.consoleUrl || ''
      }
      const file = new Blob([yaml.dump(credentials)],
```